pipelines/test/tw/ldd-check.yaml

```diff
@@ -22,7 +22,12 @@ inputs:
 pipeline:
   - name: "check for missing library dependencies using ldd"
     runs: |
+      if [ -z "${{inputs.files}}" ] && [ -z "${{inputs.packages}}" ]; then
+          packages="${{context.name}}"
```

Comment:
This probably right. 
If we're only ever passing the single package, then we can just make the default be context.name and even drop support for files. 

If needed, we can at another pipeline that calls ldd-check with files. Then the usage is super short. 

Uses: tw/ldd-check

Or

Uses:ldd-check-files
  With:
    Files: ...

(Forgive the phone added capitalization)